ecosystem/contributors.html

```diff
@@ -35,27 +35,28 @@ <h2 class="lead">The central place for PyTorch contributors to stay up-to-date w
 
     {% include past_issues.html %}
 
-    <div class="marketo-contain container-fluid">
-      <!-- START CONTENT -->
-      <div class="row col-f-6 marketo-left">
-        <div class="contributor-row">
-          <h2>Newsletter</h2>
-          <p>Follow the contributors newsletter for curated news from across the PyTorch developer community</p>
-          <a id="issue-toggle" class="contributors-button" href="#pytorch-1">View Issues</a>
-          <!-- <div class="contributor-row">
-            <h2>Sign up for the newsletter</h2>
-            <div class="input-group mb-3">
-              <input type="text" class="form-control" placeholder="email" aria-label="email" aria-describedby="email">
-              <div class="input-group-append">
-                <button id ="submit" class="contributor-submit btn btn-outline-secondary" type="button">Submit</button>
-              </div>
-            </div>
-          </div> -->
+    <div class="two-column-row">
+      <div class="content-left">
+        <h2>Newsletter Sign Up</h2>
+        <p>Follow the contributors newsletter for curated news from across the PyTorch developer community</p>
+        <div class="contributor-form">
+          <form id="contributorForm" action="http://cl.s10.exct.net/DEManager.aspx" name="subscribeForm" method="post">
```

Comment:
This should be https

Reply:
@suraj813 We had to switch to Mailchimp. This is outdated. Please see latest commit to this file.